shiny-ci-cd.Rmd

```diff
@@ -113,7 +173,6 @@ deploy:
     branch: master
```

Comment:
maybe somewhere we should add a note about the master branch being now called main if you create a new repo on GitHub

Reply:
Resolved as belongs to same topic of adding more comments to these files in general.